docs/source/release_notes/release-notes.rst

```diff
@@ -84,21 +82,30 @@ develop
          - OkHttp clients (created with ``FeignOkHttpClients``) will no longer silently retry connections.
            We have already implemented retries, including retries from connection failures, at the Feign level in ``FailoverFeignTarget``.
            If you require silent retry, please contact the AtlasDB team.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/TODO>`__)
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2197>`__)
+
+    *    - |userbreak|
```

Comment:
Well of course it's `improved`, why else would we do it? :)

...added. I had to add the 0.50.0 header anyway.